engines/default/engine.js

```diff
@@ -0,0 +1,32 @@
+import utils from './utils.js'
```

Comment:
This may be my OO-crazy speaking, but can we make Engine be a class with two required public methods, additionalParameters() and generate()?